packages/components/src/SidePanel/SidePanel.scss

```diff
@@ -55,7 +55,13 @@ $toggle-button-padding: $padding-small;
 		padding-top: 0;
 		padding-bottom: 0;
 		padding-left: $toggle-button-padding;
+		background: $list-item-reverse-color;
 		text-align: left;
+		position: sticky;
+		top: 0;
+		left: 0;
+		right: 0;
+		z-index: 10;
```

Comment:
why do you need z-index?

Reply:
<img width="266" alt="image" src="https://user-images.githubusercontent.com/26482371/65252459-458b9c00-daf9-11e9-8b72-55796cf77980.png">
because the toggle button is before the action list

Reply:
Oh no ok, I saw that it's because `<ul>` has an absolute position with simplebar :+1: